app/models/user.rb

```diff
@@ -5,11 +5,11 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable
 
   has_one_attached :icon
-  validate :image_type_allowed
+  validate :image_type_can_be_accepted
 
   private
 
-  def image_type_allowed
+  def image_type_can_be_accepted
```

Comment:
バリデーションはcanよりもmustやshouldのような強い制約を課す助動詞がよく使われますね。
あとはvalidationなので、acceptedよりもvalidの方がいいかも？

```suggestion
  def image_type_should_be_valid
```

とか

```suggestion
  def image_type_should_have_valid_content_type
```

とか

Reply:
should_be_validがしっくりくるのでそちらに修正しました。